src/core/tests/util/graph_comparator.hpp

```diff
@@ -458,7 +460,9 @@ class Storage : private AttributeStorage<MemoryChunk>,
                storage<SubGraphOpInputDescription>().get_attributes_number() +
                storage<SubGraphOpOutputDescription>().get_attributes_number() +
                storage<ov::op::util::FrameworkNodeAttrs>().get_attributes_number() +
-               storage<std::shared_ptr<ngraph::Variable>>().get_attributes_number();
+               storage<std::shared_ptr<ngraph::Variable>>().get_attributes_number() +
```

Comment:
Please use the same namespace `ov` for PartialShape and Dimension in same file.

Reply:
Done.